mypy/test/testcheck.py

```diff
@@ -221,7 +228,8 @@ def run_case_once(self, testcase: DataDrivenTestCase,
             output = testcase.output2.get(incremental_step, [])
         else:
             raise AssertionError()
-
+        # BASED: inject sussy new line
+        output = [it.replace("$NL", "\n") for it in output]
```

Comment:
what this means

Reply:
oh its for errors with newllines in thghem, maybe add a comment saying that? or can you put `\n` directly in the test files? or nah

Reply:
There was absolutely no possible way to put it in the test data., I'll add a comment